code/game/turfs/simulated.dm

```diff
@@ -51,7 +51,16 @@
 				return
 			if(TURF_WET_LUBE)
 				M.slip(0, 7, null, (SLIDE|GALOSHES_DONT_HELP))
+				return
+			if(TURF_WET_ICE)
+				M.slip(0, 4, null, (SLIDE|NO_SLIP_WHEN_WALKING))
+				return
+
 
 /turf/simulated/ChangeTurf(var/path)
 	. = ..()
 	smooth_icon_neighbors(src)
+
+turf/simulated/MakeSlippery()
+	if(TURF_WET_ICE)

```

Comment:
You sure that this is what you want to do? This is just `if(3)` which is always true.